question_thirteen.py

```diff
@@ -0,0 +1,42 @@
+from functools import reduce
+
+class Number:
+    def __init__(self, numbers):
+        self.numbers = numbers
+    
+    def get(self):
+        return self.numbers
+    
+    def change_original_values(self, func: lambda i: i): 
+        new_numbers = []
+        new_numbers = list(map(lambda i: i*2, self.numbers))
+        return new_numbers
+    
+    def filter_values(self, filter_func: lambda x: x):
+
+        filtered_numbers = [] 
+        filtered_numbers = list(filter(lambda i: (i % 2 == 0), self.numbers))
```

Comment:
The use of `filter_func` is not just that we keep there. We have to use that here. We can pass any function that will be applied on the list. Please update it. See the above comment.